tensorly/metrics/tests/test_entropy.py

```diff
@@ -170,11 +170,9 @@ def test_cp_vonneumann_entropy_mixed_state():
         / 2.0
```

Comment:
I am confused about what `test_cp_vonneumann_entropy_mixed_state()` is trying to do. It seems like this test is correctly erroring with a singular matrix error, since the true rank of the tensor is 1.

Reply:
I have no idea either... thought you were the one who wrote it !